src/bin/bindgen.rs

```diff
@@ -217,6 +217,16 @@ pub fn main() {
 
     let mut bind_args: Vec<_> = env::args().collect();
 
+    let clang_version = clang_version();
+
+    if cfg!(feature = "llvm_stable") {
+        assert_eq!(clang_version, "3.8.0");
+    } else {
+        assert_eq!(clang_version, "3.9.0");
```

Comment:
That makes sense. Provided it's actually desirable to parse what clang_getClangVersion gives in the first place, I could just extract the first two numbers of the semvar and drop anything else that comes after it.